test/integration/addons_test.go

```diff
@@ -76,7 +76,7 @@ func readLineWithTimeout(b *bufio.Reader, timeout time.Duration) (string, error)
 
 func testDashboard(t *testing.T) {
 	t.Parallel()
-	minikubeRunner := NewMinikubeRunner(t)
+	minikubeRunner := NewMinikubeRunner(t, "")
```

Comment:
Passing an empty string to a function is a code smell, as the meaning of the string is completely mysterious to readers. Let's not do that. Three alternative ideas come to mind:

- Perhaps `NewMinikubeRunner` should randomly generate the profile name. It gets populated in the return object anyways, so that clients can retrieve it if they need it. This may make test cleanup trickier, but makes parallelism easier.
- NewMinikubeRunner could allow a config struct to be passed, where one of the options is a name.
- There could be two functions: one for a named runner, and one for a default named runner.